braph2genesis/graphs/_OrdMlBD.gen.m

```diff
@@ -0,0 +1,274 @@
+%% ¡header!
+OrdMlBD < Graph (g, ordinal multilayer binary directed graph) is a ordinal multilayer binary directed graph.
+
+%%% ¡description!
+In an ordinal multilayer binary directed (BD) graph, layers could have the different number 
+ of nodes with within-layer directed edges. Edges can be either 0 (absence of connection) 
+or 1 (existence of connection).
+All node connections are allowed between layers.
+The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.
```

Comment:
... layers could have different number 
 of nodes with within-layer directed edges. (remove THE before different)